addons/hr_holidays/tests/test_out_of_office.py

```diff
@@ -37,7 +37,7 @@ def test_leave_ooo(self):
             'request_date_to': first_leave_date_end,
         })
         first_leave.action_approve()
-        # validate a leave from 2024-06-5 (Wednesday) to 2024-06-07 (Friday)
+        # validate a leave from 2024-06-05 (Wednesday) to 2024-06-07 (Friday)
```

Comment:
Hello, I'm already fixing this one in https://github.com/odoo/odoo/pull/214021/files#diff-378dfecbfd3e916572c438a7e843891631a0823aa47dbafbdd8e8a03eeae8765

As it is in a comment, could you drop it from this PR to avoid unnecessary conflict?

Reply:
If your change is the same, git will resolve the conflict silently, won't it?

Reply:
It's not the same "git change", as the line is moved too.